addons/website_sale_stock/static/tests/tours/website_sale_stock_multilang.js

```diff
@@ -0,0 +1,52 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+
+tour.register('website_sale_stock_multilang', {
+    test: true,
+    url: '/fr/shop?search=unavailable',
+},
+[{
+    content: "Do nothing",
+    trigger: "a",
+    run: () => {}, // This is a check.
```

Comment:
?
That's probably not needed, and if it is, we really need to know why -> comment